src/Raven.Client/Json/BlittableJsonWriter.cs

```diff
@@ -203,6 +203,12 @@ private void WritePropertyValue(string propName, object value)
                 case long l:
                     _manualBlittableJsonDocumentBuilder.WriteValue(l);
                     break;
+                case int i:
```

Comment:
There are couple of others that are missing here that I can think of e.g. `uint`, `ushort` etc

Is there a chance that they were not added by purpose? cc @ayende do you recall maybe if there was such a decision?

Reply:
No such reason, we just didn't add them.

Reply:
Added more types